elasticsearch.md

```diff
@@ -525,7 +525,13 @@ In ReadonlyREST we roughly classify requests as:
 * "write": when allowed, the request changes the internal state of the cluster or the data.
 
 If a **read request** asks for a some indices they have permissions for and some indices that they do NOT have permission for, the request is **rewritten** to involve only the subset of indices they have permission for.
-This is behaviour is very useful in Kibana: **different** users can see the **same** dashboards with data from only their own indices.|
+This is behaviour is very useful in Kibana: **different** users can see the **same** dashboards with data from only their own indices.
+
+When the subset of indices is empty, it means that user are not allowed to access requested indices. In multitenancy environment we should consider two options:
```

Comment:
* What subset? Define the set and the subset in question
* Remember to use the articles: i.e. "..that the users.."
* define better the user i.e. "the authenticated Kibana user" or something like this.

Reply:
the subset is defined in section above